contracts/stake/src/contract.rs

```diff
@@ -360,6 +364,33 @@ impl StakingTrait for Staking {
         WithdrawableRewardsResponse { rewards }
     }
 
+    fn update_config(env: Env, config: Config) -> Result<Config, ContractError> {
+        env.storage()
+            .instance()
+            .extend_ttl(INSTANCE_LIFETIME_THRESHOLD, INSTANCE_BUMP_AMOUNT);
+
+        let admin = get_admin(&env);
+        admin.require_auth();
+
+        save_config(&env, config.clone());
```

Comment:
Come on man, that's lazy.
Make it like in the pool - please.
https://github.com/Phoenix-Protocol-Group/phoenix-contracts/blob/main/contracts/pool/src/contract.rs#L619-L660

Reply:
updated